### PR TITLE
Add issue and PR templates

### PR DESCRIPTION
**Pull request by mpodwysocki**
Adds templates for issues for bug reports, feature requests or questions as well as a PR template

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Setup (please complete the following information):**
+ - OS: [e.g. Android 10]
+ - IDE : [e.g. Android Studio 4.0]
```
Comment (marstr) [inline .github/ISSUE_TEMPLATE/bug_report.md:30]: I don't think this is nearly as important as the version of XCode. I'd remove this field.

File: .github/ISSUE_TEMPLATE/question-query-template.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+**Query/Question**
+A clear and concise ask/query.
+
+***Why is this not a Bug or a feature Request?***
+A clear explanation of why is this not a bug or a feature request?
+
+**Setup (please complete the following information if applicable):**
+ - OS: [e.g. Android 10]
+ - IDE : [e.g. Android Studio 4.0 ]
```
Comment (marstr) [inline .github/ISSUE_TEMPLATE/question-query-template.md:18]: Same here.

File: .github/ISSUE_TEMPLATE/question-query-template.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
+---
+name: Question/Query template
+about: Use this template to file a question/query for the SDK team
+title: "[QUERY]"
+labels: ''
+assignees: ''
+
+---
+
+**Query/Question**
```
Comment (marstr) [inline .github/ISSUE_TEMPLATE/question-query-template.md:10]: Honestly, I'd add a link to Stack Overflow here that includes a notification-hubs tag that shows that we're responsive there. SO is a much better platform that GH for question/query type stuff.

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (mpodwysocki) [reply .github/ISSUE_TEMPLATE/bug_report.md:30]: Disagree as they could be using VS Code, IntelliJ, NetBeans, etc and it's important to know as much of the environment as we can to rule out other things

File: .github/ISSUE_TEMPLATE/question-query-template.md (hunk 2 above)
Comment (marstr) [reply .github/ISSUE_TEMPLATE/question-query-template.md:18]: Resolved the same as here: https://github.com/Azure/azure-notificationhubs-android/pull/111#discussion_r466600891

File: .github/ISSUE_TEMPLATE/question-query-template.md (hunk 3 above)
Comment (marstr) [reply .github/ISSUE_TEMPLATE/question-query-template.md:10]: Here's the link I'd include: https://stackoverflow.com/questions/tagged/azure-notificationhub/

Comment (marstr) [reply .github/ISSUE_TEMPLATE/question-query-template.md:11]: Fixed in 4a6b0101d68a2565b9b7a99de9230357d4eb6d46

**Pull request merged**